images/ubuntu/scripts/ubicloud/setup-runner-user.sh

```diff
@@ -100,4 +100,9 @@ ACTIONS_RUNNER_HOOK_JOB_COMPLETED=/home/runner/actions-runner/complete-hook.sh"
 echo "PATH=$PATH" >> ./actions-runner/.env
 
 mv ./actions-runner /home/runner/
+
+# Optimizes the runner process startup time
+# https://github.com/actions/runner/blob/6ca97eeb88810dc898e04d53efcd80d8b24bc4e4/src/Runner.Listener/Runner.cs#L193
```

Comment:
As it is not directly related to setting up runner user, we can move it to a new script

Reply:
I had the same thought initially, but creating a separate script file just for a one-liner felt like unnecessary overhead to maintain.